src/ApiCompat/Rules/Compat/CannotSealType.cs

```diff
@@ -18,7 +18,7 @@ public override DifferenceType Diff(IDifferences differences, ITypeDefinition im
             if (impl.IsEffectivelySealed() && !contract.IsEffectivelySealed())
             {
                 differences.AddIncompatibleDifference(this,
-                    $"Type '{impl.FullName()}' is sealed in the {Implementation} but not sealed in the {Contract}.");
+                        $"Type '{impl.FullName()}' is {(impl.IsSealed ? "effectively (has a private constructor)" : "actually (has the sealed modifier)")} sealed in the {Implementation} but not sealed in the {Contract}.");
```

Comment:
I think this is wrong.  It looks to me like when impl is actually sealed it will log the "effectively" sealed message.

Reply:
Doh, will fix.